docs/encyclopedia/nexus-endpoints.mdx

```diff
@@ -0,0 +1,37 @@
+---
+id: nexus-endpoints
+title: Nexus Endpoints
+sidebar_label: Nexus Endpoints
+description: Nexus Endpoints are reverse proxies that connect Nexus callers and handlers forwarding Nexus requests to an upstream target Namespace and Task Queue that a Worker is polling.
+toc_max_heading_level: 4
+slug: /nexus/endpoints
+keywords:
+  - nexus endpoints
+  - temporal nexus
+  - nexus
+tags:
+  - Nexus
+  - Concepts
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+
+:::
+
```

Comment:
We usually try to add a quick intro sentence before the H2 headers.

Reply:
Thanks! Will add intro sentences to each page.